SandboxiePlus/SandMan/sandman_ko.ts

```diff
@@ -2440,27 +2440,28 @@ No will choose: %2</source>
     <message>
         <location filename="SandMan.cpp" line="2883"/>
         <source>Import/Export not available, 7z.dll could not be loaded</source>
-        <translation type="unfinished"></translation>
+        <translation>가져오기/내보내기 기능을 사용할 수 없습니다, 7z.dll을 불러올 수 없습니다</translation>
     </message>
     <message>
         <location filename="SandMan.cpp" line="2884"/>
         <source>Failed to create the box archive</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>박스 압축파일을 만들지 못했습니다</translatorcomment>
+        <translation></translation>
```

Comment:
```suggestion
        <translation>박스 압축파일을 만들지 못했습니다</translation>
```

Probably an oversight, I ask for confirmation.